test/vite-node/test/hmr.test.ts

```diff
@@ -11,7 +11,7 @@ test('hmr.accept works correctly', async () => {
 
   editFile(scriptFile, content => content.replace('Hello!', 'Hello world!'))
 
-  await viteNode.waitForStderr('Hello world!')
+  await viteNode.waitForStderr('Hello world!', 40_000)
```

Comment:
Maybe it's same with https://github.com/vitejs/vite/pull/11075#discussion_r1032508251?
Doing like this might work.
```ts
const promise = viteNode.waitForStderr('Hello world!')
editFile(scriptFile, content => content.replace('Hello!', 'Hello world!'))
await promise
```

Reply:
Thank you for this information! I will try this!

Reply:
Ah, my bad, it needs to be:
```js
  const promises = [
    viteNode.waitForStderr('Hello world!'),
    viteNode.waitForStderr('Accept'),
    viteNode.waitForStdout(`[vite-node] hot updated: ${scriptFile}`)
  ]
  editFile(scriptFile, content => content.replace('Hello!', 'Hello world!'))
  await Promise.all(promises)
```